### PR TITLE
PERF-4906 Add TS test for conjunction of selective predicates

### PR DESCRIPTION
**Pull request by borosaurus**
**Jira Ticket:** https://jira.mongodb.org/browse/PERF-4906?filter=-1

**Whats Changed:**  
Adding new test for time series bucket level filters.

**Patch testing results:**  
https://spruce.mongodb.com/version/656f6d8d9ccd4e411f60db14

**Workload Submission form:**  
N/A, this is just adding a new test to existing workload.

**Related PRs:**   
N/A

### CODE REVIEW

File: src/workloads/query/TimeseriesBlockProcessing.yml [hunk 1]
```diff
@@ -202,6 +202,39 @@ Actors:
                     { $count: "count" },
                   ]
 
+# This query stresses the bucket level filter on nested fields.
```
Comment (borosaurus) [inline src/workloads/query/TimeseriesBlockProcessing.yml:205]: Verified this locally. Around 2-3% of buckets pass the bucket level filter.

Comment (parker-felix) [reply src/workloads/query/TimeseriesBlockProcessing.yml:205]: The convention in the TsbsOptimization is to include the stats of the bucket level filter in the [actor comment](https://github.com/mongodb/genny/blob/33839588affe9276eb7cc2d3d46592023dc56989/src/workloads/query/TimeseriesTsbsOptimizations.yml#L207-L208) but none of the other queries in this workload have that so its probably fine as is.

**Review by parker-felix — APPROVED** [review]
LGTM!

**Review by parker-felix — APPROVED** [review]
LGTM

**Comment by borosaurus** [comment]
evergreen merge

**Pull request merged**